tests/k6/src/test-delete-file.ts

```diff
@@ -0,0 +1,49 @@
+import {sleep, check} from 'k6';
+import {Options} from "k6/options";
+import {defaults, api} from "./lib";
+
+const files = {
+    'kb_50.jpg': open('./_files/kb_50.jpg', 'b'),
+}
+
+export let options: Options = {
+    insecureSkipTLSVerify: true,
+    iterations: 100,
+    vus: 100,
+};
+
+export const setup = (): void => {
+    console.log("setup for delete")
+    for (let vu=1; vu <= options.vus; vu++) {
+      for (let iter=0; iter < options.iterations/options.vus; iter++) {
+        const res = api.uploadFile(defaults.accounts.einstein, files['kb_50.jpg'], `deletefile_${vu}_${iter}.jpg`)
```

Comment:
its easier for later refference to keep the filenames in a shared hash. this whay we can loop over it in future test steps.

```
// 1. init code

export function setup() {
  const images: string[];
  ...
  return {images} 
}

export default function (data) {
  data.images.forEach(i => console.log(i))
}

export function teardown(data) {
  data.images.forEach(i => console.log(i))
}
```